src/Cake.Common/Tools/XUnit/XUnitRunner.cs

```diff
@@ -120,7 +115,7 @@ protected override string GetToolName()
         /// Gets the possible names of the tool executable.
         /// </summary>
         /// <returns>The tool executable name.</returns>
-        protected override IEnumerable<string> GetToolExecutableNames()
+        protected override System.Collections.Generic.IEnumerable<string> GetToolExecutableNames()
```

Comment:
@devlead while we are making this revert, can we add a using statement for `System.Collections.Generic`